### PR TITLE
Pop-up text on mob.

### PR DESCRIPTION
**Pull request by DeadLineSS13**
Add pop-up text on mob.
unknown

### CODE REVIEW

**Review by NanakoAC — CHANGES_REQUESTED** [review]
Alright, after testing it out, i've noted a few small problems:
1. It's not enabled by default, most people will never notice it without that, turn it on.
2. The line spacing between the text is a bit much, easy to make big towers, try reducing that
3. The time that it stays onscreen is too short, barely time to read it. Increase by 50%
4. Radio messages are shown there too. Generally, they should only be shown to people who were close enough to hear the message

**Comment by JakeDaBoss** [comment]
Besides what Nanako put up, did you confirm that Humans cannot see Necro's pop-ups, or does it happen for them at all?

**Comment by DeadLineSS13** [comment]
> Besides what Nanako put up, did you confirm that Humans cannot see Necro's pop-ups, or does it happen for them at all?

Nanako checked this.

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -264,15 +264,16 @@ proc/get_radio_key_from_channel(var/channel)
 	var/list/listening_clients = list()
 	for(var/mob/M in listening)
 		if(M)
-			M.hear_say(message, verb, speaking, alt_name, italics, src, speech_sound, sound_vol)
-			if(M.client)
-				listening_clients += M.client
+			if(M.hear_say(message, verb, speaking, alt_name, italics, src, speech_sound, sound_vol))
+				if(M.client)
```
Comment (NanakoAC) [inline code/modules/mob/living/say.dm]: hear_say also includes a check for client so this one is unnecessary

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**